server/tso/tso.go

```diff
@@ -265,9 +265,13 @@ func (t *TimestampOracle) GetRespTS(count uint32) (pdpb.Timestamp, error) {
 		return resp, errors.New("tso count should be positive")
 	}
 
+	failpoint.Inject("skipRetryGetTS", func() {
```

Comment:
can we add `maxRetryCount` to `TimestampOracle` to avoid using failpoint?

Reply:
Let's keep this variable unchanged inside bug fix firstly.